src/packages/src/xroad/common/base/usr/share/xroad/scripts/_restore_xroad.sh

```diff
@@ -142,7 +142,7 @@ setup_tmp_restore_dir() {
 extract_to_tmp_restore_dir () {
```

Comment:
for additional security, extract should ensure that restored files don't have the execute mode bit set (with the ability to change path, that would again create a RCE vulnerability).